lib/components/Blocks/Header/index.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { Header } from "."
+
+const meta = {
+  component: Header,
+  tags: ["autodocs"],
+  args: {
+    title: "Alba Horneros",
+    subtitle: "Product Designer",
+    src: "https://github.com/dani-moreno.png",
+    alt: "DM",
+  },
+} satisfies Meta<typeof Header>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
```

Comment:
I have the feeling that this might fit better into a `Pages` category, right? Actually, I don't really like "Blocks" and the only other element we have now is `Card` which should probably be ditched in favor of `Widget` (card shouldn't be publicly exposed).

Reply:
done